capstone/src/main/resources/static/script.js

```diff
@@ -250,6 +261,7 @@ function createIndividualProfile(data, fillForm) {
     // prefill form
     document.getElementById("ind-firstname").value = data.firstName;
     document.getElementById("ind-lastname").value = data.lastName;
+    document.getElementById("university-form-display").display = "block";
```

Comment:
I see you revealed it in this function. No change on the functionality so we can keep it this way.